### ISSUE TITLE
improving selectBox (summary)

### ISSUE THREAD

**Issue by g-sam** [smooth-operators-fac/currency-map#28]
- fix bug where clicking selects box of 0 size
- enable selection of new country inside selected box
- get rid of toggle system
- change calibration so that 0 change is pinned to middle colour
- zoom should take you to the min bounding box of all the selected countries rather than the selectbox.

**Comment by g-sam** [comment]
I wont have any time to work on this between now and thursday. I have an interview on wed that I need to prepare for (classics job -- not sure I'll take it if offered but seemed churlish to refuse interview...). actually, I'm thinking of skipping FAC for a couple of days to prepare. Therefore, get stuck in if you feel like working on these issues!

**Comment by daymos** [comment]
okey good luck with it.
will you have time to check github or should I merge my own pull requests?

**Comment by g-sam** [comment]
feel free to merge them.

**Comment by bradreeder** [comment]
Congrats, Sam. Is that for an academic position? Best of luck!

**Comment by g-sam** [comment]
Yes, though I'm going try to sell them on coding as well! Will be interesting to see how that goes down...
